.github/workflows/examples-genio.yaml

```
@@ -0,0 +1,68 @@
+# Copyright (c) 2022 Project CHIP Authors
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)